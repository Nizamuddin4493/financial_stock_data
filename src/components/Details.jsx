import React from 'react';
import { Link } from 'react-router-dom';
import stock_image from './images/stock-market.jpg'

const Details = () => (
  <>
    <div>
      <div className="detals">
      <Link to="/" className="router_link"><h3 className="details_back_arrow"> &#8617; Back</h3></Link>
        <h2>NasDaq</h2>
        <div className="details_data">
          <div>
            <div className="details_info"><h4>NasDaq:</h4><p>1275721002015</p></div>
            <div className="details_info"><h4>NasDaq:</h4><p>1275721002015</p></div>
            <div className="details_info"><h4>NasDaq:</h4><p>1275721002015</p></div>
          </div>
          <div>
            <div className="details_info"><h4>NasDaq:</h4><p>1275721002015</p></div>
            <div className="details_info"><h4>NasDaq:</h4><p>1275721002015</p></div>
            <div className="details_info"><h4>NasDaq:</h4><p>1275721002015</p></div>
          </div>
          <div>
            <div className="details_info"><h4>NasDaq:</h4><p>1275721002015</p></div>
            <div className="details_info"><h4>NasDaq:</h4><p>1275721002015</p></div>
            <div className="details_info"><h4>NasDaq:</h4><p>1275721002015</p></div>
          </div>
        </div>
        <img src={ stock_image } alt="Share Exchange" />
      </div>
    </div>
  </>
);


export default Details;