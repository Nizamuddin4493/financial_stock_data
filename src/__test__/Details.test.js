import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Details from '../components/Details';

const DetailsProvider = () => (
  <Provider store={store}>
    <Details />
  </Provider>
);

describe('renders properly the Details page', () => {
  it('renders correctly', () => {
    const component = renderer.create(<DetailsProvider />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
