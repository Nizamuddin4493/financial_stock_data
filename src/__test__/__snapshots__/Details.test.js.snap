// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly the Details page renders correctly 1`] = `<detailsProvider />`;
